source/slides/00_introduction.rst

```diff
@@ -121,6 +121,65 @@ Goals
 - Teach modern Development and Operations practices
 - Survey of important and useful techniques used 
 
+System Administration
+=====================
+
+Systems Administrators
+~~~~~~~~~~~~~~~~~~~~~~
+
+* Responsible for systems (typically servers) running code, applications,
+  and services
+
+  - Keeping applications running (they crash, sometimes a lot)
+  - Updates
+  - Security
```

Comment:
Let's merge this and the updates line above as one line so that this fits within the slide when presenting. (it cuts off otherwise)